da/test/da_test.go

```diff
@@ -124,7 +124,7 @@ func doTestRetrieve(t *testing.T, dalc da.DataAvailabilityLayerClient) {
 
 	for i := uint64(0); i < 100; i++ {
 		b := getRandomBlock(i, rand.Int()%20) //nolint:gosec
-		resp := dalc.SubmitBlock(ctx, b)
+		resp := dalc.SubmitBlocks(ctx, []*types.Block{b})
```

Comment:
can you add a test that submits multiple blocks at same da height and tests retrieve?

Reply:
There's an open PR for it that I needs debugging: https://github.com/rollkit/rollkit/pull/1085